### DISCUSSION TITLE
`Last updated` info

### DISCUSSION THREAD

**Discussion by kamieniarz** [squidfunk/mkdocs-material#5301] [Q&A] [answered]
An option to add a `last updated` information (at the top or bottom of the page) with full timestamp taken from file last modification timestamp would be great ;)

**Comment by kamilkrzyskow** [comment, marked as answer]
There is a plugin for that. 
https://github.com/timvink/mkdocs-git-revision-date-localized-plugin

The information is at the bottom, don't remember if it's possible to add at the top.

**Reply by squidfunk** [reply]
Correct! Regarding position, you can override the [`content.html`](https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/content.html) partial to change order.

**Reply by mihaigalos** [reply]
I've used this maybe naive approach to place the last changed date at the top:

```bash
cat <<EOF >docs/mkdocs/settings/overrides/content.html
{% if page.meta.source %}
    <div class="source-links">
    {% for filename in page.meta.source %}
        <span class="label label-primary">{{ filename }}</span>
    {% endfor %}
    </div>
{% endif %}

{% if page.meta.git_revision_date_localized %}
    <small>Last update: {{ page.meta.git_revision_date_localized }}</small>
{% endif %}
{% if page.meta.git_created_date_localized %}
    <small>Created: {{ page.meta.git_created_date_localized }}</small>
{% endif %}

{{ page.content }}
EOF
```
Then referenced it:
```diff
 theme:
   name: material
+  custom_dir: docs/mkdocs/settings/overrides
```
This path seems to be understood by `mkdocs serve`, but the change dates don't end up towards the top of the page (still at the bottom of it). Could the path be wrong? - I've tried using `custom_dir: mkdocs/settings/overrides` and I get an error while rebuilding the docs.

Not sure what I'm doing wrong here, can you maybe have a look please?

**Reply by mihaigalos** [reply]
Ok, seems I used the wrong file, I switched to using `docs/overrides/partials/source-file.html` and it started working.
